solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -82,6 +82,9 @@ library DKG {
         // Identifiers of candidate group members as outputted by the group
         // selection protocol.
         uint32[] members;
+        // Identifiers of candidate group members as outputted by the group
+        // selection protocol excluding IA/DQ members
+        uint32[] activeMembers;
```

Comment:
What is the gas difference in `submitDkgResult` when we need to filter out group members compared to the savings we are getting in this approach and the gas cost of the challenge?

I would like to see concrete numbers before we decide on this change, especially on 64 and 100 members group with minimum, average, and maximum possible number of misbehaving members.

It is possible this change gives great results but it is also possible that the challenge cost could get increased significantly, we need to know that for sure.

Reply:
One advantage of doing the change proposed in this PR I see is that - depending on the implementation in the challenge DKG result function in the validator - we could preserve the order of members, as returned by the sortition pool and just filter out misbehaved members instead of changing the order of group members (see the node in `setGroupMembers` function).

Reply:
Yes, I'll try to extract gas usage in a separate PR for comparison.